Git/gitignore.md

```diff
@@ -29,7 +29,9 @@ node_modules
 
 followed by running the command below, in terminal:
 
-    git config --global core.excludesfile ~/.gitignore
+```bash
```

Comment:
NIT: lets make everything similar. i see you have kept everything else as `sh`. Would prefer this as `sh ` as well

Reply:
Agree! I don't care which one just want consistency. Can we fix it outside this PR though?